draft-ietf-tls-esni.md

```diff
@@ -443,13 +455,17 @@ extensions MUST be ordered consecutively in ClientHelloInner. The list of outer
 extensions, OuterExtensions, includes those which were removed from
 EncodedClientHelloInner, in the order in which they were removed.
 
-Finally, EncodedClientHelloInner is serialized as a ClientHello structure,
-defined in {{Section 4.1.2 of RFC8446}}. Note this does not include the
-four-byte header included in the Handshake structure.
+Finally, the client determines the number of padding bytes, N, and sets the
```

Comment:
nit (here and below):
```suggestion
Finally, the client determines the number of padding bytes, length_of_padding, and sets the
```

Reply:
Well, N is referenced immediately afterwards and elsewhere we've used capital letters for random variables in prose. Note RFC8446 never utters length_of_padding in prose either. (Really what's going on here is that this is a questionably valid TLS message. The TLS message syntax is typically self-delimiting. But it's silly to add a length prefix here.)

I'll rewrite this to not reference a variable name.